elements/pfelement/src/pfelement.js

```diff
@@ -247,23 +247,22 @@ class PFElement extends HTMLElement {
       return;
     }
 
-    switch (typeof name) {
-      case "string":
-        return (
-          [...this.children].filter((child) => child.hasAttribute("slot") && child.getAttribute("slot") === name)
+    if (typeof name === "string") {
```

Comment:
@mwcz Funnily enough, I picked up this jump links branch and it needed the fix here that we talked about this morning!